src/lib/storage/table.cpp

```diff
@@ -18,75 +18,68 @@
 
 namespace opossum {
 
-Table::Table(const ChunkOffset target_chunk_size) {
-  // Implementation goes here
-  Fail("Implementation is missing.");
-}
+Table::Table(const ChunkOffset target_chunk_size) : _target_chunk_size{target_chunk_size} { create_new_chunk(); }
 
 void Table::add_column(const std::string& name, const std::string& type) {
-  // Implementation goes here
-  Fail("Implementation is missing.");
+  _column_names.push_back(name);
```

Comment:
By the documentation, it should only be possible to update the schema of empty tables. We would recommend asserting this.